pkg/kubectl/cmd/create/flags.go

```diff
@@ -69,14 +69,14 @@ func (f *PrintFlags) AddFlags(cmd *cobra.Command) {
 	}
 }
 
-func NewPrintFlags(operation string, scheme runtime.ObjectConvertor) *PrintFlags {
+func NewPrintFlags(operation string) *PrintFlags {
```

Comment:
@deads2k removing the need for a scheme arg here since the version printer is being removed, and the need for a typer in the name printer is no longer there